packages/@atjson/source-gdocs-paste/src/converter.ts

```diff
@@ -64,6 +64,24 @@ GDocsSource.defineConverterTo(OffsetSource, (doc) => {
     .set({ type: "-offset-heading" })
     .rename({ attributes: { "-gdocs-level": "-offset-level" } });
 
+  doc
+    .where({ type: "-offset-heading", attributes: { "-gdocs-align": 1 } })
+    .unset("attributes.-gdocs-align")
+    .set({ attributes: { "-offset-alignment": "center" } });
+
+  doc
+    .where({ type: "-offset-heading", attributes: { "-gdocs-align": 2 } })
+    .unset("attributes.-gdocs-align")
+    .set({ attributes: { "-offset-alignment": "end" } });
+
+  doc
+    .where({ type: "-offset-heading", attributes: { "-gdocs-align": 3 } })
+    .unset("attributes.-gdocs-align")
+    .set({ attributes: { "-offset-alignment": "justify" } });
+
+  // Remove all other alignments
+  doc.where({ type: "-offset-heading" }).unset("attributes.-gdocs-align");
```

Comment:
These conversions are clear (👍 ) but un-optimal performance-wise (😞  4 passes over all the document annotations when it could be done in 1). It may be worth it to do it in one pass?

Reply:
I discussed with @bachbui about this— we have an unfortunate pollution of our attributes namespace with attributes from other places. I think it would make a lot of sense to add support for tombstoned attributes that are safely accessible under another property. This would allow us to cleanly tuck away data that we'd like to keep around, but don't actually change the meaning of the annotation.

I'll spin this out to its own ticket, but the gist of it would be:

```ts
class Annotation<Attributes> {
  attributes: Attributes;
  /**
   * Tombstoned attributes of the annotation. These will *not* be serialized
   * when persisting to JSON, or accessible by renderers. You have been warned.
   */
  $attributes: { [key: string]: unknown };
}
```

Using this approach, we could then access the gdocs property safely:

```
doc.where({ type: "-offset-heading" }).update((heading: Heading) => {
  let align = heading.$attributes["-gdocs-align"] as AttributesOf<GDocsHeading>["align"];
  // Do the conversion
});
```

Reply:
That's one way around this but for now I don't see why this just can't be something like
```
doc.where({ type: "-gdocs-ps_hd" }).update((gdocsHeading) => {
  let { level, alignment } = gdocsHeading.attributes;
  let attributes = {};
  ...
  doc.replaceAnnotation(gdocsHeading, new Heading({ attributes, ... }));
})
```
like you have below w/ paragraphs

Reply:
poke @tim-evans

Reply:
Please make this change